librz/bin/format/elf/elf.h

```diff
@@ -127,33 +126,12 @@ typedef struct Elf_(rz_bin_elf_note_file_t) {
 }
 RzBinElfNoteFile;
 
-/// Parsed PT_NOTE of type NT_PRSTATUS
-typedef struct Elf_(rz_bin_elf_note_prstatus_t) {
-	size_t regstate_size;
-	ut8 *regstate;
-	// Hint: there is more info in NT_PRSTATUS notes that could be parsed if needed.
-}
-RzBinElfNotePrStatus;
-
-/// A single PT_NOTE entry, parsed from an ElfW(Nhdr) and associated data.
-typedef struct Elf_(rz_bin_elf_note_t) {
-	Elf_(Word) type;
-	union {
-		struct {
-			size_t files_count;
-			RzBinElfNoteFile *files;
-		} file; //< for type == NT_FILE
-		RzBinElfNotePrStatus prstatus; //< for type = NT_PRSTATUS
-	};
-}
-RzBinElfNote;
+typedef struct rz_bin_elf_note_prstatus_t {
+	size_t size;
+	ut8 *state;
```

Comment:
prstatus can have more than just the regstate, so this should stay `regstate_size`+`regstate` or similar.